### PR TITLE
Fixes - Issue #28

### PR DESCRIPTION
**Pull request by siddharthashw**
Fixes - #28 

**1. IStringConstant.cs** - removed Test cases string constants
**2. StringConstant.cs** - removed Test cases string constants and used environment variable for clientId and clientSecret
**3. IStringConstantTest.cs** - added test cases string constant
**4. StringConstantTest.cs** - added test cases string constant
**5. BaseProvider.cs** - register IStringConstantTest
**6. IntegrationBaseProvider.cs** - register IStringConstantTest
**7. ProjectModuleIntegrationTest.cs** - used StringConstantTest instead of StringConstant
**8. ProjectModuleTest.cs** - used StringConstantTest instead of StringConstant
**9. UserModuleIntegrationTest.cs** - used StringConstantTest instead of StringConstant
**10. UserModuleTest.cs** - used StringConstantTest instead of StringConstant

### CODE REVIEW

File: Promact.OAuth.Client/src/Promact.OAuth.Client.Test/IntegrationBaseProvider.cs [hunk 1]
```diff
@@ -33,8 +33,8 @@ public IntegrationBaseProvider()
             var discovery = new DiscoveryClient(_stringConstant.PromactOAuthUrl);
             discovery.Policy.RequireHttps = false;
             _discoveryClient = discovery.GetAsync().Result;
-            _client = new TokenClient(_discoveryClient.TokenEndpoint, _stringConstant.PromactOAuthClientId, 
-                _stringConstant.PromactOAuthClientSecret);
+            _client = new TokenClient(_discoveryClient.TokenEndpoint, Environment.GetEnvironmentVariable(_stringConstant.PromactOAuthClientId),
```
Comment (chintans) [inline Promact.OAuth.Client/src/Promact.OAuth.Client.Test/IntegrationBaseProvider.cs]: This file required no change

File: Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs [hunk 2]
```diff
@@ -428,7 +428,7 @@ public string PromactOAuthClientId
         {
             get
             {
-                return "OOKK4SVDYQ8XRUU";
+                return "PromactOAuthClientId";
```
Comment (chintans) [inline Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs]: Test related constants should be in Test project. here you could return Environment.GetVariable

**Review by chintans — CHANGES_REQUESTED** [review]

File: Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs (hunk 2 above)
Comment (siddharthashw) [reply Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs]: ok

File: Promact.OAuth.Client/src/Promact.OAuth.Client.Test/IntegrationBaseProvider.cs (hunk 1 above)
Comment (siddharthashw) [reply Promact.OAuth.Client/src/Promact.OAuth.Client.Test/IntegrationBaseProvider.cs]: ok

**Review by chintans — APPROVED** [review]

**Pull request merged**